Cargo.toml

```diff
@@ -16,6 +16,9 @@ license = "MIT"
 
 description = "A contiguous region of bytes, useful for I/O operations."
 
+[dependencies]
+num = "*"
```

Comment:
I figured you might be averse to adding a dependency. I can update this PR if you want by inlining `PrimInt`.